metrics_sdk/lib/opentelemetry/sdk/metrics/export/metric_reader.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Metrics
+      module Export
+        class MetricReader
+          attr_reader :metric_store
+
+          def initialize(exporter)
+            @exporter = exporter
```

Comment:
```suggestion
          def initialize
```
I *think* we don't need to expose the `metric_store` and the `exporter` isn't used. Tell me why I'm wrong.

Reply:
Removed the exporter, and as you discovered later it is being exposed to allow for registering instruments later.